index.html

```diff
@@ -51,5 +52,183 @@ <h3>LET’S CONNECT</h3>
+        </ul>
+        <button class="margin-spacer">see Project</button>
+      </div>
+      <div class="grid-item">
+        <div class="image-wrapper">
+          <img src="/images/tonic.png" alt="work-image" />
+        </div>
+
+        <h1>Tonic</h1>
+        <div class="work-details">
+          <p class="first-detail">CANOPY</p>
+          <div class="spacer margin-spacer"></div>
+          <p class="margin-spacer">Back End Dev</p>
+          <div class="spacer margin-spacer"></div>
+          <p class="margin-spacer">2015</p>
+        </div>
+        <h2>
+          A daily selection of privately personalized reads; no accounts or
+          sign-ups required.
+        </h2>
+        <ul>
+          <li>HTML</li>
+          <li class="margin-spacer">CSS</li>
+          <li class="margin-spacer">JavaScript</li>
+        </ul>
+        <button class="margin-spacer">see Project</button>
+      </div>
+    </section>
+    <section class="about-myself">
+      <h1>About Myself</h1>
+      <p>
+        Hello I’m a software developer! I can help you build a product , feature
+        or website. Look through some of my work and experience! If you like
+        what you see and have a project you need coded, don’t hesitate to
+        contact me.
+      </p>
+      <h3>LET’S CONNECT</h3>
+      <ul class="social-media">
+        <li><img src="/images/twitter-icon.svg" alt="twitter" /></li>
+        <li class="social-media-spacer">
+          <img src="/images/Linkedin icon.svg" alt="Linkedin" />
+        </li>
+        <li class="social-media-spacer">
+          <img src="/images/m.svg" alt="m" />
+        </li>
+        <li class="social-media-spacer">
+          <img src="/images/github.svg" alt="github" />
+        </li>
+        <li class="social-media-spacer">
+          <img src="/images/chill.svg" alt="chill" />
+        </li>
+      </ul>
+      <button class="margin-spacer">Get My resume</button>
+      <div class="languages-subsection">
+        <div class="sub-header">
+          <h2>Languages</h2>
+          <img src="/images/Icon - Chevron - Down - Larger.png" alt="" />
+        </div>
+        <div class="lang-planner">
+          <img src="/images/Ellipse 1.png" alt="" />
+          <p>Javascript</p>
+        </div>
+        <div class="lang-planner">
+          <img src="/images/html.png" alt="" />
+          <p>HTML</p>
+        </div>
+        <div class="lang-planner">
+          <img src="/images/css.png" alt="" />
+          <p>CSS</p>
+        </div>
+      </div>
+      <div class="frameworks-subsection">
+        <div class="sub-header">
+          <h2>Frameworks</h2>
+          <img src="/images/Icon - Chevron - Right - Larger.png" alt="" />
+        </div>
+      </div>
+      <div class="skills-subsection">
+        <div class="sub-header">
+          <h2>skills</h2>
+          <img src="/images/Icon - Chevron - Right - Larger.png" alt="" />
+        </div>
+      </div>
```

Comment:
Please kindly use a `ul` tag as the container in `line 202`  instead of a `div` and each of the languages should be in a `li` tag for a good semantic and a great UX interpretation 😄